src/post/actionButtons/PayoutLabel.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
+import numeral from 'numeral';
 import Icon from '../../widgets/Icon';
 
-const PayoutLabel = ({ value, onClick }) =>
-  <a onClick={onClick && onClick} >
-    <span className="hidden-xs">
-      <Icon name="attach_money" sm />
-    </span>
-    {value}
-  </a>;
+const PayoutLabel = ({ post, onClick }) => {
+  const payout = parseFloat(post.total_payout_value)
+    + parseFloat(post.total_pending_payout_value)
+    + parseFloat(post.curator_payout_value);
+  const maxAcceptedPayout = parseFloat(post.max_accepted_payout);
+  return (
+    <a onClick={onClick && onClick}>
```

Comment:
there is no need for this. If you don't send `onClick` from the main component react will automatically remove this listener so it's safe to use: `onClick={onClick}`